test/unit/presenters/formats/edition_format_presenter_test.rb

```diff
@@ -171,5 +172,15 @@ def result
         refute result.has_key?(:access_limited)
       end
     end
+
+    should "[:format]" do
```

Comment:
this is poorly named, as it's testing the `rendering_app` attribute of the rendered hash.